models/topics.models.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection");
+
+function selectTopics() {
+  return db
+    .query(
+      `
+        SELECT * FROM topics
+        `
+    )
+    .then(({ rows }) => {
+      return rows;
+    })
+    .catch((err) => {
```

Comment:
.catch are generally only in the controllers, the model should not be concerned with how errors are presented